user/howto/launch-dask-gateway-cluster.md

```diff
@@ -0,0 +1,225 @@
+# Launch a dask-gateway cluster
+
+This guide shows you how to launch a Dask gateway cluster for parallel and distributed computing.
+
+:::{contents}
+:depth: 2
+:local:
+:::
+
+## What is Dask Gateway?
+
+Dask Gateway allows users to launch clusters for scaling computations efficiently with more CPU and memory on cloud resources, without requiring direct access to the underlying Kubernetes backend of the 2i2c hub. Configuration, such as efficient cluster resourcing, authentication and security settings, is automatically handled for users to provide a consistent user experience across the hub.
+
+:::{admonition} What's the difference between Dask and Dask Gateway
+:class: note
+When using Dask on a local machine, you will often instantiate a cluster using
+```python
+from dask.distributed import LocalCluster
+cluster = LocalCluster()
+client = cluster.get_client()
+```
+For a 2i2c Dask hub, we instantiate a cluster with Dask Gateway that requests cloud resources with Kubernetes from a shared, centrally managed cluster environment.
+:::
+
+## Usage
+
+This section roughly follows the [Dask Gateway Docs – Usage](https://gateway.dask.org/usage.html) together with the [Training on Large Datasets](https://examples.dask.org/machine-learning/training-on-large-datasets.html) example.
+
+:::{warning}
+Avoid large, long-running, idle clusters, which consume cloud computing resources and costs. Only use a cluster when you need it and shutdown when not in use.
+:::
+
+### Connect to a dask-gateway server
+
+Create a gateway client to communicate with the dask-gateway server.
+
+```python
+from dask_gateway import Gateway
+gateway = Gateway()  # Use values stored in your local configuration (recommended)
+```
+
+:::{note}
+Leave the argument of the `Gateway()` constructor empty to use the default configuration for your 2i2c Dask hub which automatically supplies parameters such as `address`, `proxy_address`, `auth`, etc.
```

Comment:
```suggestion
Leave the argument of the `Gateway()` constructor empty to use the default configuration for your 2i2c Dask hub which automatically supplies parameters such as `address`, `proxy_address`, `auth`, etc. via environment variables.
```

It could perhaps be better to not add this technical detail to keep things briefer?

Reply:
```suggestion
Leave the argument of the `Gateway()` constructor empty to use the default configuration for your 2i2c Dask hub (parameters such as `address`, `proxy_address`, `auth`, etc. will be automatically supplied to the `Gateway()` object)
```

I am going to leave this technical detail in our docs because it addresses the confusion in a FreshDesk ticket that I believe stems from the [Dask Gateway docs](https://gateway.dask.org/usage.html#connect-to-a-dask-gateway-server) telling users that `Gateway()` typically takes these parameters.